sequencer/dbmanager.go

```diff
@@ -178,18 +195,50 @@ func (d *dbManager) GetWIPBatch(ctx context.Context) (*WipBatch, error) {
 	}
 
 	wipBatch := &WipBatch{
-		batchNumber:  lastBatch.BatchNumber,
-		coinbase:     lastBatch.Coinbase,
-		accInputHash: lastBatch.AccInputHash,
-		// initialStateRoot: lastBatch.StateRoot,
+		batchNumber:    lastBatch.BatchNumber,
+		coinbase:       lastBatch.Coinbase,
+		accInputHash:   lastBatch.AccInputHash,
```

Comment:
You need to init here also the `initialAccInputHash` and the `InitialStateRoot` if there is a previously closed batch. (turned out we need those fields for L2Reorgs.